### PR TITLE
chore(datasets): version all the data and update the paths

### PR DESCRIPTION
**Pull request by gforsyth**
Does what it says on the tin.

Also adding a `local_setup.py` script so that attendees who want to run things locally can pre-download the necessary data without running afoul of conference wifi.

### CODE REVIEW

File: local_setup.py
Comment (cpcloud) [inline local_setup.py:1]: Are we invoking this script somewhere?

Comment (cpcloud) [reply local_setup.py:1]: If not, seems like we're likely to forget to update it if/when the quarto version changes.

Comment (gforsyth) [reply local_setup.py:1]: Not yet -- I'm planning to reference it in an email to tutorial attendees, advising them to run it beforehand if they want to do the tutorial on their local machine

Comment (gforsyth) [reply local_setup.py:1]: we will _definitely_ forget to update it, but it will suffice for the pycon tutorial

Comment (cpcloud) [reply local_setup.py:1]: SGTM.

Do we need to mention verbally in the tutorial that we won't be debugging local installations for people? (I'm assuming we're not)

**Review by cpcloud — APPROVED** [review]

File: local_setup.py
Comment (gforsyth) [reply local_setup.py:1]: I think we'll mention that in the pre-tutorial email blast -- that folks are welcome to install things locally but that we don't have bandwidth to troubleshoot installation issues

**Pull request merged**